### PR TITLE
add calibrate resource error for 7.1

### PR DESCRIPTION
**Pull request by CabinfeverB**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
ref https://github.com/pingcap/tidb/issues/45166, we should add error description for 7.1.0. When 7.1.1 release, we should change this because of https://github.com/pingcap/tidb/pull/45167 will fixed it.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-calibrate-resource.md [hunk 1]
```diff
@@ -94,6 +94,13 @@ CALIBRATE RESOURCE START_TIME '2023-04-18 08:00:00' DURATION '60m';
 ERROR 1105 (HY000): The workload in selected time window is too low, with which TiDB is unable to reach a capacity estimation; please select another time window with higher workload, or calibrate resource by hardware instead
 ```
 
+估算容量功能需要监控指标数据，包括 `resource_manager_resource_unit`、`process_cpu_usage`、`tikv_cpu_quota`、`tidb_server_maxprocs`。当对应监控数据为空，会有对应监控项名称的报错。当完全没有负载时，`resource_manager_resource_unit` 为空，也会报告此错误。
```
Comment (hfxsd) [inline sql-statements/sql-statement-calibrate-resource.md:97]:
```suggestion
估算容量功能需要监控指标数据，包括 `resource_manager_resource_unit`、`process_cpu_usage`、`tikv_cpu_quota`、`tidb_server_maxprocs`。当对应监控数据为空，会有对应监控项名称的报错，如下面例子所示。当完全没有负载时，`resource_manager_resource_unit` 为空，也会报告此错误。
```

**Review by hfxsd — APPROVED** [review]

File: dashboard/dashboard-resource-manager.md [hunk 2]
```diff
@@ -59,6 +59,8 @@ summary: 介绍如何使用 TiDB Dashboard 的资源管控页面查看资源管
 
     - 如果时间窗口范围内的负载过低，会报错 `Error 1105 (HY000): The workload in selected time window is too low, with which TiDB is unable to reach a capacity estimation; please select another time window with higher workload, or calibrate resource by hardware instead`。
 
+    - 估算容量功能需要监控指标数据，包括 `resource_manager_resource_unit`、`process_cpu_usage`、`tikv_cpu_quota`、`tidb_server_maxprocs`。如果对应监控数据为空，会有对应监控项名称的报错，如 `Error 1105 (HY000): metrics ‘resource_manager_resource_unit’ is empty`。当完全没有负载时，`resource_manager_resource_unit` 为空，也会报告此错误。此外，由于 TiKV 未在 macOS 上监控 CPU 使用率，所以不支持估算容量功能。
```
Comment (HuSharp) [inline dashboard/dashboard-resource-manager.md:62]: macOS 这一句需要放在 `根据实际负载估算容量` 的下方以作为提醒吗？

File: sql-statements/sql-statement-calibrate-resource.md [hunk 3]
```diff
@@ -94,6 +96,20 @@ CALIBRATE RESOURCE START_TIME '2023-04-18 08:00:00' DURATION '60m';
...
 ```
 
+估算容量功能需要监控指标数据，包括 `resource_manager_resource_unit`、`process_cpu_usage`、`tikv_cpu_quota`、`tidb_server_maxprocs`。当对应监控数据为空，会有对应监控项名称的报错，如下面例子所示。当完全没有负载时，`resource_manager_resource_unit` 为空，也会报告此错误。
+
+```sql
+CALIBRATE RESOURCE START_TIME '2023-04-18 08:00:00' DURATION '60m';
+Error 1105 (HY000): metrics 'resource_manager_resource_unit' is empty
+```
+
+由于 TiKV 未在 macOS 上监控 CPU 使用率，会报错提供无相应监控数据。
```
Comment (HuSharp) [inline sql-statements/sql-statement-calibrate-resource.md:106]: 或者 `会报错未提供相应监控数据` ?
```suggestion
由于 TiKV 未在 macOS 上监控 CPU 使用率，会报告相应监控数据为空的错误。
```

**Review by HuSharp — APPROVED** [review]
LGTM!

File: sql-statements/sql-statement-calibrate-resource.md [hunk 4]
```diff
@@ -94,6 +98,20 @@ CALIBRATE RESOURCE START_TIME '2023-04-18 08:00:00' DURATION '60m';
 ERROR 1105 (HY000): The workload in selected time window is too low, with which TiDB is unable to reach a capacity estimation; please select another time window with higher workload, or calibrate resource by hardware instead
 ```
 
+估算容量功能需要监控指标数据，包括 `resource_manager_resource_unit`、`process_cpu_usage`、`tikv_cpu_quota`、`tidb_server_maxprocs`。当对应监控数据为空，会有对应监控项名称的报错，如下面例子所示。当完全没有负载时，`resource_manager_resource_unit` 为空，也会报告此错误。
```
Comment (hfxsd) [inline sql-statements/sql-statement-calibrate-resource.md:101]:
```suggestion
估算容量功能需要监控指标数据，包括 `resource_manager_resource_unit`、`process_cpu_usage`、`tikv_cpu_quota`、`tidb_server_maxprocs`。当对应监控数据为空，会出现对应监控项名称的报错，如下面例子所示。当完全没有负载时，`resource_manager_resource_unit` 为空，也会出现此错误。
```

File: sql-statements/sql-statement-calibrate-resource.md [hunk 5]
```diff
@@ -94,6 +98,20 @@ CALIBRATE RESOURCE START_TIME '2023-04-18 08:00:00' DURATION '60m';
...
 ```
 
+估算容量功能需要监控指标数据，包括 `resource_manager_resource_unit`、`process_cpu_usage`、`tikv_cpu_quota`、`tidb_server_maxprocs`。当对应监控数据为空，会出现对应监控项名称的报错，如下面例子所示。当完全没有负载时，`resource_manager_resource_unit` 为空，也会出现此错误。
+
+```sql
+CALIBRATE RESOURCE START_TIME '2023-04-18 08:00:00' DURATION '60m';
+Error 1105 (HY000): metrics 'resource_manager_resource_unit' is empty
+```
+
+由于 TiKV 未在 macOS 上监控 CPU 使用率，会报告相应监控数据为空的错误。
```
Comment (hfxsd) [inline sql-statements/sql-statement-calibrate-resource.md:108]:
```suggestion
由于 TiKV 未在 macOS 上监控 CPU 使用率，在 macOS 上执行 `CALIBRATE RESOURCE` 会出现以下错误：
```

File: sql-statements/sql-statement-calibrate-resource.md [hunk 6]
```diff
@@ -94,6 +98,20 @@ CALIBRATE RESOURCE START_TIME '2023-04-18 08:00:00' DURATION '60m';
...
 ```
 
+估算容量功能需要监控指标数据，包括 `resource_manager_resource_unit`、`process_cpu_usage`、`tikv_cpu_quota`、`tidb_server_maxprocs`。当对应监控数据为空，会出现对应监控项名称的报错，如下面例子所示。当完全没有负载时，`resource_manager_resource_unit` 为空，也会出现此错误。
+
+```sql
+CALIBRATE RESOURCE START_TIME '2023-04-18 08:00:00' DURATION '60m';
+Error 1105 (HY000): metrics 'resource_manager_resource_unit' is empty
+```
+
+由于 TiKV 未在 macOS 上监控 CPU 使用率，在 macOS 上执行 `CALIBRATE RESOURCE` 会出现以下错误：
```
Comment (CabinfeverB) [inline sql-statements/sql-statement-calibrate-resource.md:108]: @hfxsd 这里不能这么改，只是不支持"根据实际负载估算容量功能"

Comment (CabinfeverB) [inline sql-statements/sql-statement-calibrate-resource.md:108]:
```suggestion
由于 TiKV 未在 macOS 上监控 CPU 使用率，在 macOS 上使用根据实际负载估算容量功能会出现以下错误：
```

File: dashboard/dashboard-resource-manager.md [hunk 7]
```diff
@@ -59,6 +59,8 @@ summary: 介绍如何使用 TiDB Dashboard 的资源管控页面查看资源管
 
     - 如果时间窗口范围内的负载过低，会报错 `Error 1105 (HY000): The workload in selected time window is too low, with which TiDB is unable to reach a capacity estimation; please select another time window with higher workload, or calibrate resource by hardware instead`。
 
+    - 根据实际负载估算容量功能需要监控指标数据，包括 `resource_manager_resource_unit`、`process_cpu_usage`、`tikv_cpu_quota`、`tidb_server_maxprocs`。如果对应监控数据为空，会出现对应监控项名称的报错，如 `Error 1105 (HY000): metrics 'resource_manager_resource_unit' is empty`。当完全没有负载时，`resource_manager_resource_unit` 为空，也会出现此错误。此外，由于 TiKV 未在 macOS 上监控 CPU 使用率，所以不支持根据实际负载估算容量功能, 会报错 `ERROR 1105 (HY000): metrics 'process_cpu_usage' is empty`。
```
Comment (ran-huang) [inline dashboard/dashboard-resource-manager.md:62]:
```suggestion
    - 根据实际负载估算容量功能需要监控指标数据，包括 `resource_manager_resource_unit`、`process_cpu_usage`、`tikv_cpu_quota`、`tidb_server_maxprocs`。如果对应监控数据为空，会出现对应监控项名称的报错，如 `Error 1105 (HY000): metrics 'resource_manager_resource_unit' is empty`。当完全没有负载时，`resource_manager_resource_unit` 为空，也会出现此错误。此外，由于 TiKV 未在 macOS 上监控 CPU 使用率，所以不支持根据实际负载估算容量功能，会报错 `ERROR 1105 (HY000): metrics 'process_cpu_usage' is empty`。
```

File: sql-statements/sql-statement-calibrate-resource.md [hunk 8]
```diff
@@ -31,6 +31,10 @@ TiDB 提供两种预估方式：
 - 时间窗口范围为 10 分钟至 24 小时。
 - 在预估的时间窗口内，TiDB 与 TiKV 的 CPU 利用率不能过低，否则无法进行容量估算。
 
+> **注意：**
+>
+> 由于 TiKV 未在 macOS 上监控 CPU 使用率，所以不支持在 macOS 上使用根据实际负载估算容量功能。
```
Comment (ran-huang) [inline sql-statements/sql-statement-calibrate-resource.md:36]:
```suggestion
> 由于 TiKV 未在 macOS 上监控 CPU 使用率，所以不支持在 macOS 上根据实际负载估算容量。
```

File: sql-statements/sql-statement-calibrate-resource.md [hunk 9]
```diff
@@ -94,6 +98,20 @@ CALIBRATE RESOURCE START_TIME '2023-04-18 08:00:00' DURATION '60m';
 ERROR 1105 (HY000): The workload in selected time window is too low, with which TiDB is unable to reach a capacity estimation; please select another time window with higher workload, or calibrate resource by hardware instead
 ```
 
+估算容量功能需要监控指标数据，包括 `resource_manager_resource_unit`、`process_cpu_usage`、`tikv_cpu_quota`、`tidb_server_maxprocs`。当对应监控数据为空，会出现对应监控项名称的报错，如下面例子所示。当完全没有负载时，`resource_manager_resource_unit` 为空，也会出现此错误。
```
Comment (ran-huang) [inline sql-statements/sql-statement-calibrate-resource.md:101]:
```suggestion
估算容量功能需要监控指标数据，包括 `resource_manager_resource_unit`、`process_cpu_usage`、`tikv_cpu_quota`、`tidb_server_maxprocs`。当对应监控数据为空时，会出现对应监控项名称的报错，如下面例子所示。当完全没有负载时，`resource_manager_resource_unit` 为空，也会出现此错误。
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**